### ISSUE TITLE
False positive when version number is > 9 (Spring Framework)

### ISSUE THREAD

**Issue by dadoonet** [OSSIndex/ossindex-maven-plugin#21]
I'm now getting a failure on my project when applying this plugin.

https://ossindex.sonatype.org/vuln/098bb0f6-44d1-489e-888f-65477990b8b1

When I look at my dependency tree, I have:

```
[INFO] --- maven-dependency-plugin:3.1.1:tree (default-cli) @ fscrawler-tika ---
[INFO] fr.pilato.elasticsearch.crawler:fscrawler-tika:jar:2.6-SNAPSHOT
[INFO] +- fr.pilato.elasticsearch.crawler:fscrawler-framework:jar:2.6-SNAPSHOT:compile
[INFO] |  +- commons-io:commons-io:jar:2.5:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.9.6:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.9.6:compile
[INFO] |  +- com.fasterxml.jackson.datatype:jackson-datatype-jsr310:jar:2.9.6:compile
[INFO] |  +- com.fasterxml.jackson.dataformat:jackson-dataformat-xml:jar:2.9.6:compile
[INFO] |  |  +- com.fasterxml.jackson.module:jackson-module-jaxb-annotations:jar:2.9.6:compile
[INFO] |  |  +- org.codehaus.woodstox:stax2-api:jar:3.1.4:compile
[INFO] |  |  \- com.fasterxml.woodstox:woodstox-core:jar:5.0.3:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.9.6:compile
[INFO] |  +- org.apache.logging.log4j:log4j-core:jar:2.11.1:compile (optional) 
[INFO] |  |  \- org.apache.logging.log4j:log4j-api:jar:2.11.1:compile (optional) 
[INFO] |  +- org.apache.logging.log4j:log4j-1.2-api:jar:2.11.1:compile (optional) 
[INFO] |  +- org.apache.logging.log4j:log4j-slf4j-impl:jar:2.11.1:compile (optional) 
[INFO] |  +- org.apache.logging.log4j:log4j-jcl:jar:2.11.1:compile (optional) 
[INFO] |  |  \- commons-logging:commons-logging:jar:1.2:compile
[INFO] |  +- org.apache.logging.log4j:log4j-jul:jar:2.11.1:compile (optional) 
[INFO] |  \- org.fusesource.jansi:jansi:jar:1.17.1:compile (optional) 
[INFO] +- fr.pilato.elasticsearch.crawler:fscrawler-beans:jar:2.6-SNAPSHOT:compile
[INFO] +- fr.pilato.elasticsearch.crawler:fscrawler-settings:jar:2.6-SNAPSHOT:compile
[INFO] +- org.apache.tika:tika-parsers:jar:1.18:compile
[INFO] |  +- org.apache.tika:tika-core:jar:1.18:compile
[INFO] |  +- org.gagravarr:vorbis-java-tika:jar:0.8:compile
[INFO] |  +- com.healthmarketscience.jackcess:jackcess:jar:2.1.10:compile
[INFO] |  |  \- commons-lang:commons-lang:jar:2.6:compile
[INFO] |  +- com.healthmarketscience.jackcess:jackcess-encrypt:jar:2.1.4:compile
[INFO] |  +- org.tallison:jmatio:jar:1.2:compile
[INFO] |  +- org.apache.james:apache-mime4j-core:jar:0.8.1:compile
[INFO] |  +- org.apache.james:apache-mime4j-dom:jar:0.8.1:compile
[INFO] |  +- org.apache.commons:commons-compress:jar:1.16.1:compile
[INFO] |  |  \- org.objenesis:objenesis:jar:2.6:compile
[INFO] |  +- org.tukaani:xz:jar:1.8:compile
[INFO] |  +- org.brotli:dec:jar:0.1.2:compile
[INFO] |  +- commons-codec:commons-codec:jar:1.10:compile
[INFO] |  +- org.apache.pdfbox:pdfbox:jar:2.0.9:compile
[INFO] |  |  \- org.apache.pdfbox:fontbox:jar:2.0.9:compile
[INFO] |  +- org.apache.pdfbox:pdfbox-tools:jar:2.0.9:compile
[INFO] |  +- org.apache.pdfbox:jempbox:jar:1.8.13:compile
[INFO] |  +- org.bouncycastle:bcmail-jdk15on:jar:1.54:compile
[INFO] |  |  \- org.bouncycastle:bcpkix-jdk15on:jar:1.54:compile
[INFO] |  +- org.bouncycastle:bcprov-jdk15on:jar:1.60:compile
[INFO] |  +- org.apache.poi:poi:jar:3.17:compile
[INFO] |  |  \- org.apache.commons:commons-collections4:jar:4.1:compile
[INFO] |  +- org.apache.poi:poi-scratchpad:jar:3.17:compile
[INFO] |  +- org.apache.poi:poi-ooxml:jar:3.17:compile
[INFO] |  |  +- org.apache.poi:poi-ooxml-schemas:jar:3.17:compile
[INFO] |  |  |  \- org.apache.xmlbeans:xmlbeans:jar:2.6.0:compile
[INFO] |  |  \- com.github.virtuald:curvesapi:jar:1.04:compile
[INFO] |  +- org.ccil.cowan.tagsoup:tagsoup:jar:1.2.1:compile
[INFO] |  +- org.ow2.asm:asm:jar:5.0.4:compile
[INFO] |  +- com.googlecode.mp4parser:isoparser:jar:1.1.18:compile
[INFO] |  +- com.drewnoakes:metadata-extractor:jar:2.10.1:compile
[INFO] |  |  \- com.adobe.xmp:xmpcore:jar:5.1.3:compile
[INFO] |  +- de.l3s.boilerpipe:boilerpipe:jar:1.1.0:compile
[INFO] |  +- com.rometools:rome:jar:1.5.1:compile
[INFO] |  |  \- com.rometools:rome-utils:jar:1.5.1:compile
[INFO] |  +- org.gagravarr:vorbis-java-core:jar:0.8:compile
[INFO] |  +- com.googlecode.juniversalchardet:juniversalchardet:jar:1.0.3:compile
[INFO] |  +- org.codelibs:jhighlight:jar:1.0.2:compile
[INFO] |  +- com.pff:java-libpst:jar:0.8.1:compile
[INFO] |  +- com.github.junrar:junrar:jar:0.7:compile
[INFO] |  +- org.apache.commons:commons-exec:jar:1.3:compile
[INFO] |  +- org.apache.opennlp:opennlp-tools:jar:1.8.4:compile
[INFO] |  +- com.googlecode.json-simple:json-simple:jar:1.1.1:compile
[INFO] |  +- com.github.openjson:openjson:jar:1.0.10:compile
[INFO] |  +- com.google.code.gson:gson:jar:2.8.1:compile
[INFO] |  +- org.slf4j:slf4j-api:jar:1.7.24:compile
[INFO] |  +- org.slf4j:jul-to-slf4j:jar:1.7.24:compile
[INFO] |  +- org.slf4j:jcl-over-slf4j:jar:1.7.24:compile
[INFO] |  +- org.jsoup:jsoup:jar:1.11.2:compile
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.4:compile
[INFO] |  |  \- org.apache.httpcomponents:httpcore:jar:4.4.7:compile
[INFO] |  +- org.apache.httpcomponents:httpmime:jar:4.5.4:compile
[INFO] |  +- org.apache.commons:commons-csv:jar:1.0:compile
[INFO] |  +- org.apache.sis.core:sis-utility:jar:0.8:compile
[INFO] |  |  \- javax.measure:unit-api:jar:1.0:compile
[INFO] |  +- org.apache.sis.storage:sis-netcdf:jar:0.8:compile
[INFO] |  |  +- org.apache.sis.storage:sis-storage:jar:0.8:compile
[INFO] |  |  |  \- org.apache.sis.core:sis-feature:jar:0.8:compile
[INFO] |  |  \- org.apache.sis.core:sis-referencing:jar:0.8:compile
[INFO] |  +- org.apache.sis.core:sis-metadata:jar:0.8:compile
[INFO] |  +- org.opengis:geoapi:jar:3.0.1:compile
[INFO] |  +- edu.usc.ir:sentiment-analysis-parser:jar:0.1:compile
[INFO] |  +- org.apache.uima:uimafit-core:jar:2.2.0:compile
[INFO] |  |  +- org.springframework:spring-core:jar:3.2.16.RELEASE:compile
[INFO] |  |  +- org.springframework:spring-context:jar:3.2.16.RELEASE:compile
[INFO] |  |  |  +- org.springframework:spring-aop:jar:3.2.16.RELEASE:compile
[INFO] |  |  |  |  \- aopalliance:aopalliance:jar:1.0:compile
[INFO] |  |  |  \- org.springframework:spring-expression:jar:3.2.16.RELEASE:compile
[INFO] |  |  \- org.springframework:spring-beans:jar:3.2.16.RELEASE:compile
[INFO] |  +- org.apache.uima:uimaj-core:jar:2.9.0:compile
[INFO] |  +- org.jdom:jdom2:jar:2.0.6:compile
[INFO] |  \- org.apache.pdfbox:jbig2-imageio:jar:3.0.0:compile
[INFO] +- org.xerial:sqlite-jdbc:jar:3.23.1:compile
[INFO] +- com.levigo.jbig2:levigo-jbig2-imageio:jar:2.0:compile (optional) 
[INFO] +- com.github.jai-imageio:jai-imageio-core:jar:1.4.0:compile (optional) 
[INFO] +- com.github.jai-imageio:jai-imageio-jpeg2000:jar:1.3.0:compile (optional) 
[INFO] +- org.apache.tika:tika-langdetect:jar:1.18:compile
[INFO] |  +- com.optimaize.languagedetector:language-detector:jar:0.6:compile
[INFO] |  |  +- net.arnx:jsonic:jar:1.2.11:compile
[INFO] |  |  \- com.intellij:annotations:jar:12.0:compile
[INFO] |  \- com.google.guava:guava:jar:17.0:compile
[INFO] +- com.jcraft:jsch:jar:0.1.54:compile
[INFO] +- fr.pilato.elasticsearch.crawler:fscrawler-test-framework:jar:2.6-SNAPSHOT:test
[INFO] |  +- org.apache.lucene:lucene-test-framework:jar:7.3.1:test
[INFO] |  |  +- org.apache.lucene:lucene-codecs:jar:7.3.1:test
[INFO] |  |  \- org.apache.lucene:lucene-core:jar:7.3.1:test
[INFO] |  +- org.hamcrest:hamcrest-all:jar:1.3:test
[INFO] |  +- junit:junit:jar:4.12:test
[INFO] |  |  \- org.hamcrest:hamcrest-core:jar:1.3:test
[INFO] |  \- com.carrotsearch.randomizedtesting:randomizedtesting-runner:jar:2.6.3:test
[INFO] \- fr.pilato.elasticsearch.crawler:fscrawler-test-documents:jar:2.6-SNAPSHOT:test
```

The Spring versions are `3.2.16.RELEASE` which is after `3.2.8` but apparently this is reported as a previous version.

Full build logs here: https://travis-ci.org/dadoonet/fscrawler/builds/419748117

Extract:

```
[INFO] --- ossindex-maven-plugin:1.0.0:audit (audit-dependencies) @ fscrawler-tika ---
[INFO] Checking for vulnerabilities:
[INFO]   aopalliance:aopalliance:jar:1.0:compile
[INFO]   com.googlecode.juniversalchardet:juniversalchardet:jar:1.0.3:compile
[INFO]   org.apache.pdfbox:pdfbox-tools:jar:2.0.9:compile
[INFO]   com.github.jai-imageio:jai-imageio-core:jar:1.4.0:compile
[INFO]   fr.pilato.elasticsearch.crawler:fscrawler-beans:jar:2.6-SNAPSHOT:compile
[INFO]   com.fasterxml.jackson.module:jackson-module-jaxb-annotations:jar:2.9.6:compile
[INFO]   org.xerial:sqlite-jdbc:jar:3.23.1:compile
[INFO]   org.bouncycastle:bcpkix-jdk15on:jar:1.54:compile
[INFO]   org.objenesis:objenesis:jar:2.6:compile
[INFO]   org.slf4j:jul-to-slf4j:jar:1.7.24:compile
[INFO]   com.googlecode.mp4parser:isoparser:jar:1.1.18:compile
[INFO]   com.levigo.jbig2:levigo-jbig2-imageio:jar:2.0:compile
[INFO]   com.googlecode.json-simple:json-simple:jar:1.1.1:compile
[INFO]   org.apache.james:apache-mime4j-core:jar:0.8.1:compile
[INFO]   org.apache.httpcomponents:httpclient:jar:4.5.4:compile
[INFO]   com.rometools:rome:jar:1.5.1:compile
[INFO]   org.apache.tika:tika-langdetect:jar:1.18:compile
[INFO]   org.apache.pdfbox:jempbox:jar:1.8.13:compile
[INFO]   org.apache.pdfbox:pdfbox:jar:2.0.9:compile
[INFO]   commons-logging:commons-logging:jar:1.2:compile
[INFO]   org.fusesource.jansi:jansi:jar:1.17.1:compile
[INFO]   com.optimaize.languagedetector:language-detector:jar:0.6:compile
[INFO]   com.healthmarketscience.jackcess:jackcess:jar:2.1.10:compile
[INFO]   org.apache.logging.log4j:log4j-slf4j-impl:jar:2.11.1:compile
[INFO]   com.intellij:annotations:jar:12.0:compile
[INFO]   org.apache.logging.log4j:log4j-jul:jar:2.11.1:compile
[INFO]   edu.usc.ir:sentiment-analysis-parser:jar:0.1:compile
[INFO]   org.apache.logging.log4j:log4j-api:jar:2.11.1:compile
[INFO]   org.jdom:jdom2:jar:2.0.6:compile
[INFO]   org.jsoup:jsoup:jar:1.11.2:compile
[INFO]   org.hamcrest:hamcrest-all:jar:1.3:test
[INFO]   org.apache.poi:poi-ooxml-schemas:jar:3.17:compile
[INFO]   org.apache.poi:poi:jar:3.17:compile
[INFO]   org.apache.logging.log4j:log4j-core:jar:2.11.1:compile
[INFO]   fr.pilato.elasticsearch.crawler:fscrawler-test-framework:jar:2.6-SNAPSHOT:test
[INFO]   org.apache.opennlp:opennlp-tools:jar:1.8.4:compile
[INFO]   net.arnx:jsonic:jar:1.2.11:compile
[INFO]   com.fasterxml.jackson.core:jackson-annotations:jar:2.9.6:compile
[INFO]   org.apache.httpcomponents:httpcore:jar:4.4.7:compile
[INFO]   commons-lang:commons-lang:jar:2.6:compile
[INFO]   org.apache.uima:uimafit-core:jar:2.2.0:compile
[INFO]   com.fasterxml.jackson.core:jackson-databind:jar:2.9.6:compile
[INFO]   org.apache.commons:commons-collections4:jar:4.1:compile
[INFO]   org.apache.commons:commons-exec:jar:1.3:compile
[INFO]   com.carrotsearch.randomizedtesting:randomizedtesting-runner:jar:2.6.3:test
[INFO]   org.springframework:spring-core:jar:3.2.16.RELEASE:compile
[INFO]   commons-codec:commons-codec:jar:1.10:compile
[INFO]   org.apache.commons:commons-csv:jar:1.0:compile
[INFO]   org.gagravarr:vorbis-java-core:jar:0.8:compile
[INFO]   com.pff:java-libpst:jar:0.8.1:compile
[INFO]   org.tukaani:xz:jar:1.8:compile
[INFO]   org.apache.commons:commons-compress:jar:1.16.1:compile
[INFO]   com.adobe.xmp:xmpcore:jar:5.1.3:compile
[INFO]   org.apache.poi:poi-ooxml:jar:3.17:compile
[INFO]   org.apache.pdfbox:jbig2-imageio:jar:3.0.0:compile
[INFO]   org.springframework:spring-context:jar:3.2.16.RELEASE:compile
[INFO]   com.github.jai-imageio:jai-imageio-jpeg2000:jar:1.3.0:compile
[INFO]   org.apache.sis.core:sis-metadata:jar:0.8:compile
[INFO]   org.bouncycastle:bcprov-jdk15on:jar:1.60:compile
[INFO]   org.apache.sis.core:sis-referencing:jar:0.8:compile
[INFO]   org.apache.sis.storage:sis-storage:jar:0.8:compile
[INFO]   org.apache.tika:tika-core:jar:1.18:compile
[INFO]   com.github.openjson:openjson:jar:1.0.10:compile
[INFO]   com.google.guava:guava:jar:17.0:compile
[INFO]   com.rometools:rome-utils:jar:1.5.1:compile
[INFO]   org.springframework:spring-expression:jar:3.2.16.RELEASE:compile
[INFO]   org.apache.james:apache-mime4j-dom:jar:0.8.1:compile
[INFO]   javax.measure:unit-api:jar:1.0:compile
[INFO]   org.opengis:geoapi:jar:3.0.1:compile
[INFO]   com.github.virtuald:curvesapi:jar:1.04:compile
[INFO]   org.ow2.asm:asm:jar:5.0.4:compile
[INFO]   org.springframework:spring-aop:jar:3.2.16.RELEASE:compile
[INFO]   org.apache.pdfbox:fontbox:jar:2.0.9:compile
[INFO]   com.fasterxml.jackson.datatype:jackson-datatype-jsr310:jar:2.9.6:compile
[INFO]   com.fasterxml.jackson.dataformat:jackson-dataformat-xml:jar:2.9.6:compile
[INFO]   org.codelibs:jhighlight:jar:1.0.2:compile
[INFO]   org.hamcrest:hamcrest-core:jar:1.3:test
[INFO]   com.jcraft:jsch:jar:0.1.54:compile
[INFO]   org.springframework:spring-beans:jar:3.2.16.RELEASE:compile
[INFO]   com.google.code.gson:gson:jar:2.8.1:compile
[INFO]   org.apache.lucene:lucene-test-framework:jar:7.3.1:test
[INFO]   org.bouncycastle:bcmail-jdk15on:jar:1.54:compile
[INFO]   commons-io:commons-io:jar:2.5:compile
[INFO]   de.l3s.boilerpipe:boilerpipe:jar:1.1.0:compile
[INFO]   org.apache.httpcomponents:httpmime:jar:4.5.4:compile
[INFO]   org.apache.tika:tika-parsers:jar:1.18:compile
[INFO]   org.brotli:dec:jar:0.1.2:compile
[INFO]   org.apache.sis.core:sis-utility:jar:0.8:compile
[INFO]   org.slf4j:slf4j-api:jar:1.7.24:compile
[INFO]   org.slf4j:jcl-over-slf4j:jar:1.7.24:compile
[INFO]   fr.pilato.elasticsearch.crawler:fscrawler-framework:jar:2.6-SNAPSHOT:compile
[INFO]   com.github.junrar:junrar:jar:0.7:compile
[INFO]   com.drewnoakes:metadata-extractor:jar:2.10.1:compile
[INFO]   org.gagravarr:vorbis-java-tika:jar:0.8:compile
[INFO]   fr.pilato.elasticsearch.crawler:fscrawler-settings:jar:2.6-SNAPSHOT:compile
[INFO]   org.apache.uima:uimaj-core:jar:2.9.0:compile
[INFO]   org.apache.logging.log4j:log4j-jcl:jar:2.11.1:compile
[INFO]   org.apache.sis.core:sis-feature:jar:0.8:compile
[INFO]   com.fasterxml.woodstox:woodstox-core:jar:5.0.3:compile
[INFO]   org.ccil.cowan.tagsoup:tagsoup:jar:1.2.1:compile
[INFO]   org.apache.xmlbeans:xmlbeans:jar:2.6.0:compile
[INFO]   org.apache.sis.storage:sis-netcdf:jar:0.8:compile
[INFO]   org.apache.lucene:lucene-core:jar:7.3.1:test
[INFO]   com.fasterxml.jackson.core:jackson-core:jar:2.9.6:compile
[INFO]   org.apache.logging.log4j:log4j-1.2-api:jar:2.11.1:compile
[INFO]   org.codehaus.woodstox:stax2-api:jar:3.1.4:compile
[INFO]   org.apache.lucene:lucene-codecs:jar:7.3.1:test
[INFO]   junit:junit:jar:4.12:test
[INFO]   org.tallison:jmatio:jar:1.2:compile
[INFO]   fr.pilato.elasticsearch.crawler:fscrawler-test-documents:jar:2.6-SNAPSHOT:test
[INFO]   com.healthmarketscience.jackcess:jackcess-encrypt:jar:2.1.4:compile
[INFO]   org.apache.poi:poi-scratchpad:jar:3.17:compile
[INFO] Exclude coordinates: []
[INFO] Exclude vulnerability identifiers: []
[INFO] CVSS-score threshold: 0.0
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] FSCrawler .......................................... SUCCESS [  0.956 s]
[INFO] FSCrawler Test Framework ........................... SUCCESS [  3.373 s]
[INFO] FSCrawler Framework ................................ SUCCESS [  1.483 s]
[INFO] FSCrawler Settings ................................. SUCCESS [  5.105 s]
[INFO] FSCrawler Test Documents ........................... SUCCESS [  0.125 s]
[INFO] FSCrawler Elasticsearch Client ..................... SUCCESS [  3.622 s]
[INFO] FSCrawler Beans .................................... SUCCESS [  2.528 s]
[INFO] FSCrawler Tika ..................................... FAILURE [ 10.926 s]
[INFO] FSCrawler Crawlers ................................. SKIPPED
[INFO] FSCrawler Abstract Crawler ......................... SKIPPED
[INFO] FSCrawler Crawlers: FS ............................. SKIPPED
[INFO] FSCrawler Crawlers: SSH ............................ SKIPPED
[INFO] FSCrawler Core ..................................... SKIPPED
[INFO] FSCrawler Rest Server .............................. SKIPPED
[INFO] FSCrawler Integration Tests ........................ SKIPPED
[INFO] FSCrawler Cli ...................................... SKIPPED
[INFO] FSCrawler ZIP Distribution ......................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 28.736 s
[INFO] Finished at: 2018-08-23T16:56:22Z
[INFO] Final Memory: 53M/976M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.sonatype.ossindex.maven:ossindex-maven-plugin:1.0.0:audit (audit-dependencies) on project fscrawler-tika: Detected 1 vulnerable components:
[ERROR]   org.apache.logging.log4j:log4j-core:jar:2.11.1:compile; https://ossindex.sonatype.org/component/pkg:maven/org.springframework/spring-core@3.2.16.RELEASE
[ERROR]     * [CVE-2014-1904]  Improper Neutralization of Input During Web Page Generation ("Cross-site Scripting") (4.3); https://ossindex.sonatype.org/vuln/098bb0f6-44d1-489e-888f-65477990b8b1
```

**Comment by ken-duck** [comment]
Thanks for the report. We will look into the version comparison code and resolve this as soon as possible. Sorry for the inconvenience.

**Comment by ken-duck** [comment]
Oh, I misread which plugin this was.

Not sure why this is happening, but I would suggest trying the "ossindex-maven" plugin instead, as this plugin is deprecated. The new plugin supports the OSS Index v3 API which provides many more vulnerabilities, and is being actively maintained.

You can find some information about migrating here: https://github.com/OSSIndex/ossindex-maven-plugin/blob/master/README.md

The new plugin's homepage is here: https://sonatype.github.io/ossindex-maven/

And the sources are here: https://github.com/sonatype/ossindex-maven

**Comment by dadoonet** [comment]
I tried:

* `org.sonatype.ossindex.maven:ossindex-maven-plugin:3.0.0`: same error
* `org.sonatype.ossindex.maven:ossindex-maven:3.0.0`: This does not seem to be available (and the [documentation](https://sonatype.github.io/ossindex-maven/) has not been updated if I'm not mistaken). Actually this has been published as a `pom` artifact: https://repo1.maven.org/maven2/org/sonatype/ossindex/maven/ossindex-maven/3.0.0/

```
[ERROR] Plugin org.sonatype.ossindex.maven:ossindex-maven:3.0.0 or one of its dependencies could not be resolved: Failure to find org.sonatype.ossindex.maven:ossindex-maven:jar:3.0.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
```

**Comment by ken-duck** [comment]
Hmm. Odd. I was not able to try this out yet today, but I will endeavour to reproduce the problem over the weekend and will get back to you as soon as possible.

**Comment by ken-duck** [comment]
Hi. We are trying a few theories, but meanwhile we were wondering if you were running against an open source project that we could test against as well?

**Comment by dadoonet** [comment]
Awesome. Thanks!

**Comment by dadoonet** [comment]
I'm sure the test was passing previously so my guess is that something changed on the server side. I mean that I don't think that the maven plugin is where the bug suddenly appeared.

**Comment by jdillon** [comment]
BTW ossindex-maven-plugin 3.0.1 is released, and should hit central in a few hours or so.  We did find one bug which I noticed when looking at the version range issue which was a bug in the client with how it was correlating component-report to maven artifact.

**Comment by jdillon** [comment]
Also @dadoonet you can find the latest documentation here: https://sonatype.github.io/ossindex-maven/maven-plugin/ and the reason why you had dependency resolution problems was because 'ossindex-maven' is a POM not a maven-plugin.  The correct artifactId is `ossindex-maven-plugin`.

**Comment by ken-duck** [comment]
Jason has mentioned the plugin fix that was performed. There is also a data problem on the server side that I will be resolving this weekend, so you will continue to see the vulnerability against your spring framework `3.2.16` version until that fix has been completed.

**Comment by dadoonet** [comment]
I upgraded to ossindex plugin 3.0.1 and I'm getting now:

```
ERROR] Failed to execute goal org.sonatype.ossindex.maven:ossindex-maven-plugin:3.0.1:audit (default-cli) on project fscrawler-tika: Detected 1 vulnerable components:
[ERROR]   org.springframework:spring-core:jar:3.2.16.RELEASE:compile; https://ossindex.sonatype.org/component/pkg:maven/org.springframework/spring-core@3.2.16.RELEASE
[ERROR]     * [CVE-2018-1270]  Improperly Implemented Security Check for Standard (9.8); https://ossindex.sonatype.org/vuln/9a3de118-b038-49ed-9af7-533210c9d85f
[ERROR]     * [CVE-2018-1271]  Improper Limitation of a Pathname to a Restricted Directory ("Path Traversal") (5.9); https://ossindex.sonatype.org/vuln/580d61c3-20df-4bb8-99c3-36c89e0d7550
[ERROR]     * [CVE-2018-1272]  Permissions, Privileges, and Access Controls (7.5); https://ossindex.sonatype.org/vuln/aa7190e3-4c47-42d6-82f6-afaf1da5762e
```

This looks better and a real report.

Note that I finally updated then my project dependency to Tika 1.19 which removed the Spring dependency so I don't have anymore this security error.

So I believe I can close this issue now. Thanks.
